examples/complete/main.tf

```diff
@@ -190,3 +190,42 @@ module "secret_manager_imported_cert" {
   imported_cert_private_key  = resource.tls_private_key.key.private_key_pem
   imported_cert_intermediate = resource.tls_self_signed_cert.ca_cert.cert_pem
 }
+
+##############################################################################
+# Example working with service credentials secret
+##############################################################################
+
+# create a COS instance to create the service credential for
+module "cloud_object_storage" {
+  source                 = "terraform-ibm-modules/cos/ibm"
+  version                = "7.1.4"
+  resource_group_id      = module.resource_group.resource_group_id
+  region                 = var.region
+  cos_instance_name      = "${var.prefix}-cos"
+  cos_tags               = var.resource_tags
+  bucket_name            = "${var.prefix}-bucket"
+  retention_enabled      = false # disable retention for test environments - enable for stage/prod
+  kms_encryption_enabled = false
+}
+
+#create a service authorization between Secrets Manager and the target service (COS)
+resource "ibm_iam_authorization_policy" "policy" {
+  source_service_name         = "secrets-manager"
+  target_service_name         = "cloud-object-storage"
+  target_resource_instance_id = module.cloud_object_storage.cos_instance_guid
+  roles                       = ["Key Manager"]
+}
```

Comment:
question for maybe the playback next week - should the module support creating the auth policy if its going to support creating the service credentials

Reply:
@alex-reiff do you remember what we decided to do here?

Reply:
I believe we said that this is fine for the example. We don't have any functionality to add it in the module itself.